<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>newtm</class>
 <widget class="QWidget" name="newtm">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>308</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Tilemap</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>151</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Format:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="listWidget">
      <property name="editTriggers">
       <set>QAbstractItemView::SelectedClicked</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>220</y>
     <width>261</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Name:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="nameEdit">
      <property name="maxLength">
       <number>15</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="saveBox">
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
      </property>
      <property name="centerButtons">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>261</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Tiles Resources:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="listFiles"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>140</y>
     <width>261</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Tile Width / Height (INT):</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignRight">
       <widget class="QLineEdit" name="tileWHEdit">
        <property name="maxLength">
         <number>3</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSlider" name="horizontalSlider">
      <property name="minimum">
       <number>10</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="pageStep">
       <number>2</number>
      </property>
      <property name="value">
       <number>32</number>
      </property>
      <property name="tracking">
       <bool>true</bool>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="invertedAppearance">
       <bool>false</bool>
      </property>
      <property name="invertedControls">
       <bool>false</bool>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksBothSides</enum>
      </property>
      <property name="tickInterval">
       <number>0</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
